cumulus/parachains/integration-tests/emulated/tests/people/people-rococo/src/tests/reap_identity.rs

```diff
@@ -0,0 +1,138 @@
+		github: Data::Raw(b"xcm-test".to_vec().try_into().unwrap()),
+		discord: Data::Raw(b"xcm-test#0042".to_vec().try_into().unwrap()),
+	}
+}
+
+#[test]
+fn reap_identity() {
+	let mut bal_before_relaychain: Balance = 0_u128;
+	let mut bal_after_relaychain: Balance = 0_u128;
+
+	let mut bal_before_parachain: Balance = 0_u128;
+	let mut bal_after_parachain: Balance = 0_u128;
+
+	let identity_relaychain = identity_relay();
+	let identity_parachain = identity_parachain();
+
+	// Set identity and Subs on Relay Chain
+	RococoRelay::execute_with(|| {
+		type RuntimeEvent = <RococoRelay as Chain>::RuntimeEvent;
+
+		bal_before_relaychain =
+			<RococoRelay as RococoRelayPallet>::Balances::free_balance(RococoRelaySender::get());
+
+		// 1. Set identity on Relay Chain
+		assert_ok!(<RococoRelay as RococoRelayPallet>::Identity::set_identity(
+			rococo_runtime::RuntimeOrigin::signed(RococoRelaySender::get()),
+			Box::new(identity_relaychain)
+		));
+		assert_expected_events!(
+			RococoRelay,
+			vec![
+				RuntimeEvent::Identity(pallet_identity::Event::IdentitySet { .. }) => {},
+			]
+		);
+
+		// 2. Set sub-identity on Relay Chain
+		assert_ok!(<RococoRelay as RococoRelayPallet>::Identity::set_subs(
+			rococo_runtime::RuntimeOrigin::signed(RococoRelaySender::get()),
+			vec![(RococoRelaySender::get(), Data::Raw(vec![40; 1].try_into().unwrap()))],
+		));
+		assert_expected_events!(
+			RococoRelay,
+			vec![
+				RuntimeEvent::Balances(pallet_balances::Event::Reserved { .. }) => {},
+				RuntimeEvent::Identity(pallet_identity::Event::IdentitySet { .. }) => {},
+				RuntimeEvent::Balances(pallet_balances::Event::Reserved { .. }) => {},
+			]
+		);
+
+		bal_after_relaychain =
+			<RococoRelay as RococoRelayPallet>::Balances::free_balance(RococoRelaySender::get());
+	});
+
+	// Set identity and Subs on Parachain with Zero deposit
+	PeopleRococo::execute_with(|| {
+		type RuntimeEvent = <PeopleRococo as Chain>::RuntimeEvent;
+
+		bal_before_parachain =
+			<PeopleRococo as PeopleRococoPallet>::Balances::free_balance(PeopleRococoSender::get());
+
+		// 3. Set identity on Parachain with zero deposit
+		assert_ok!(<PeopleRococo as PeopleRococoPallet>::Identity::set_identity_no_deposit(
+			&PeopleRococoSender::get(),
+			identity_parachain.clone()
+		));
+
+		// 4. Set sub-identity on Parachain
+		assert_ok!(<PeopleRococo as PeopleRococoPallet>::Identity::set_subs(
+			people_rococo_runtime::RuntimeOrigin::signed(PeopleRococoSender::get()),
+			vec![(PeopleRococoSender::get(), Data::Raw(vec![40; 1].try_into().unwrap()))],
+		));
+		assert_expected_events!(
+			PeopleRococo,
+			vec![
+				RuntimeEvent::Balances(pallet_balances::Event::Reserved { .. }) => {},
+			]
+		);
+
+		bal_after_parachain =
+			<PeopleRococo as PeopleRococoPallet>::Balances::free_balance(PeopleRococoSender::get());
+		let fees = bal_before_parachain - bal_after_parachain;
+		assert_eq!(bal_before_parachain - fees, bal_after_parachain);
```

Comment:
There are no fees in this context AFAIK, only deposits. But they should all be zero.